.github/workflows/workflow_dispatch.yml

```diff
@@ -3,6 +3,17 @@ name: Workflow on demand (workflow_dispatch)
 # Workflow on demand: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs
 # How to run: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
 on:
+  workflow_call:
+    inputs:
+      from_workflow_call:
+        description: 'To know the trigger was workflow_call'
+        required: false
+        default: true
+        type: boolean
+      tags:
+        description: 'Test scenario tags'
+        required: false
+        type: boolean
```

Comment:
Default is false
https://github.com/pgreze/github-actions-howto/actions/runs/3709981516/jobs/6289089960#step:2:63